src/Storefront/Controller/VerificationHashController.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Controller;
+
+use Shopware\Core\Framework\Routing\Annotation\RouteScope;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+use Shopware\Storefront\Exception\VerificationHashNotConfiguredException;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @RouteScope(scopes={"storefront"})
+ */
+class VerificationHashController extends AbstractController
+{
+    /**
+     * @var SystemConfigService
+     */
+    private $systemConfigService;
+
+    public function __construct(SystemConfigService $systemConfigService)
+    {
+        $this->systemConfigService = $systemConfigService;
+    }
+
+    /**
+     * @Route("/sw-domain-hash.html", name="frontend.verification-hash.load", methods={"GET"}, defaults={"allow_maintenance"=true})
+     *
+     * @throws VerificationHashNotConfiguredException
+     */
+    public function load(SalesChannelContext $context): Response
+    {
+        $salesChannelId = $context->getSalesChannel()->getId();
+        $verificationHash = $this->systemConfigService->get('core.store.verificationHash', $salesChannelId);
```

Comment:
```suggestion
        $verificationHash = $this->systemConfigService->get('core.store.verificationHash');
```
Only the instance has to be verified

Reply:
But I can change it per saleschannel, am I right?

Reply:
The shop domain in account is based on store license host. This is per installation and not per sales channel

Reply:
Oh, I checked the admin interface. There is no sales channel switch. Nice